client/src/Components/Pages/Dashboard/PersonalInfo/index.js

```diff
@@ -47,64 +47,65 @@ class PersonalInfo extends Component {
     const fav_subjects = this.state.fav_subjects || [];
 
     return (
-      <div className="PersonalInfo">
-        <h2>Personal Info</h2>
-        <span className="info">
-          Name:
-          {' '}
-          {first_name}
-          {' '}
-          {last_name}
-        </span>
-        <span className="info">
-Date of birth:
-          {' '}
-          {birthdate}
-        </span>
-        <span className="info">
-City:
-          {' '}
-          {city}
-        </span>
-        <span className="info">
-Favourit subject:
+      <div>
+        <div className="personal1 PersonalInfo">
+          <h2>Personal Info</h2>
+          <span className="info">
+            {' '}
+            {first_name}
+            {' '}
+            {last_name}
+          </span>
+
+          <span className="info">
+            From:
+            {' '}
+            {city}
+          </span>
+          <span className="info">Date of birth:</span>
+          <span className="info">
+            {' '}
+            {birthdate}
+          </span>
+          <span className="info">Favourit subject: </span>
           {
             fav_subjects && fav_subjects.map((val, key) => (
-              <span>{val.value}</span>
+              <ul>
+                <li>{val.value}</li>
+                <li>{val.value}</li>
+                <li>{val.value}</li>
```

Comment:
@donia98 Do you need just one of these?  It looks like it's printing the same thing three times.

Reply:
No need for that I fixed that, maybe she was trying to see how it looks for the style

Reply:
Ah ok!